cmd/container-disk-v1alpha/entry-point.sh

```diff
@@ -40,6 +40,26 @@ IMAGE_EXTENSION=$(echo $IMAGE_PATH | sed  -n -e 's/^.*\.\(.*\)$/\1/p')
 mkdir -p $COPY_PATH
 echo $IMAGE_NAME | grep -q -e "raw" -e "qcow2"
 if [ $? -ne 0 ]; then
+	output=$(qemu-img info $IMAGE_NAME --output=json)
```

Comment:
I think the problem is here is with the `$IMAGE_NAME`

Can you change the line to
```bash
output=$(/usr/bin/qemu-img info $IMAGE_PATH --output=json)
```

Reply:
Done.

Reply:
@slintes I think the test are better now can you take a look?

Reply:
yes, much better, thanks for having a look at it! Let's wait for the last 2 test lanes, will merge then.